specification/communication/data-plane/MediaComposition/preview/2022-07-16-preview1/examples/MediaComposition_CreateMediaComposition.json

```diff
@@ -0,0 +1,111 @@
+{
+  "parameters": {
+    "endpoint": "https://contoso.westus.communications.azure.com",
+    "api-version": "2022-07-16-preview1",
```

Comment:
`API version should be a date in YYYY-MM-DD format, optionally suffixed with '-preview'.`
You should increment the preview date and NOT add an integer to the end of the existing version identifier.